src/infra/docs/rustc-ci.md

```diff
@@ -96,31 +96,27 @@ same time, even if there is a normal PR in progress.
 
 ## Which branches we test
 
-Our builders are defined in `src/ci/azure-pipelines/`, and they depend on the
-branch used for the build. Each job is configured in one of the top `.yml`
-files.
+Our builders are defined in [`src/ci/github-actions/ci.yml`].
+
+[`src/ci/github-actions/ci.yml`]: https://github.com/rust-lang/rust/blob/master/src/ci/github-actions/ci.yml
 
 ### PR builds
 
 All the commits pushed in a PR run a limited set of tests: a job containing a
 bunch of lints plus a cross-compile check build to Windows mingw (without
-producing any artifacts) and the `x86_64-gnu-llvm-6.0` non-dist builder. Those
-two builders are enough to catch most of the common errors introduced in a PR,
-but they don’t cover other platforms at all. Unfortunately it would take too
-many resources to run the full test suite for each commit on every PR.
+producing any artifacts) and the `x86_64-gnu-llvm-##` non-dist builder (where
+`##` is the *system* LLVM version we are currently testing). Those two
+builders are enough to catch most of the common errors introduced in a PR, but
+they don’t cover other platforms at all. Unfortunately it would take too many
+resources to run the full test suite for each commit on every PR.
 
-Additionally, if the PR changes submodules the `x86_64-gnu-tools` non-dist
+Additionally, if the PR changes certain tools, the `x86_64-gnu-tools` non-dist
```

Comment:
These days I think we also run it on some standard library changes, to catch miri breakage?

Reply:
Oh, indeed. I added a note about that, too.